### PR TITLE
Blog-App: Forms

### PR DESCRIPTION
**Pull request by degisew**
### Blog-App: Forms
**In this PR, I implemented the following features:**
- Create a method called current_user in your ApplicationController
- Creates a Post on behalf of the current_user
- Create a comment on behalf of the current_user
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -4,9 +4,28 @@ def index
...
+  end
+
   def show
     @current_user = User.find_by!(id: params[:user_id])
     @posts = Post.where(author_id: @current_user)
     @post = @posts.find(params[:id])
   end
+
+  def create
+    # @post = Post.new(params.require(:new_post).permit(:title, :text))
```
Comment (BertrandConxy) [inline app/controllers/posts_controller.rb]: - [ ] Please remove the above-commented codes in order to keep your codebase clean and more understandable  😃

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @degisew, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used

## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (degisew) [reply app/controllers/posts_controller.rb]: Okay, Thanks a lot. I will fix it.

**Review by uwadonat — APPROVED** [review]
## Hi @degisew 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉
### Highlights
- Professional readme file ✔️
- Descriptive pull request message ✔️
- Well-organized and clean project  ✔️
209362895-9762f2f8-13cf-4ce0-90b6-a672c4f04196

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by degisew** [comment]
Thanks @uwadonat.

**Pull request merged**